src/helpers/DateUtil.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import { getUnixTime } from "date-fns";
+
+function getCurrentDate() {
+    const date = new Date();
+    const nowUtc = Date.UTC(
+        date.getUTCFullYear(),
+        date.getUTCMonth(),
+        date.getUTCDate(),
+        date.getUTCHours(),
+        date.getUTCMinutes(),
+        date.getUTCSeconds()
+    );
+    return new Date(nowUtc);
+}
+
+function getCurrentUnixTime(): String {
```

Comment:
I think it's more visibility if we'll didn't wrap `getUnixTime(getCurrentDate())` to function

Reply:
Just call it as is when it needed